assets/js/modules/analytics-4/components/dashboard/ActivationBanner/SetupBanner.js

```diff
@@ -31,13 +31,20 @@ import {
 	PropertySelect,
 	UseSnippetSwitch,
 } from '../../../../analytics-4/components/common';
-import useExistingTagEffect from '../../../../analytics-4/hooks/useExistingTagEffect';
-import { MODULES_ANALYTICS_4 } from '../../../datastore/constants';
+import ErrorNotice from '../../../../../components/ErrorNotice';
+import SpinnerButton from '../../../../../components/SpinnerButton';
+import {
+	MODULES_ANALYTICS_4,
+	PROPERTY_CREATE,
+} from '../../../datastore/constants';
 import { CORE_USER } from '../../../../../googlesitekit/datastore/user/constants';
+import useExistingTagEffect from '../../../../analytics-4/hooks/useExistingTagEffect';
 import { getBannerDismissalExpiryTime } from '../../../utils/banner-dismissal-expiry';
-const { useSelect } = Data;
+const { useDispatch, useSelect } = Data;
 
 export default function SetupBanner( { onCTAClick } ) {
```

Comment:
I'd suggest renaming `onCTAClick` to something that better represents what it's used for in the component - how about `onSubmitSuccess` ?